### PR TITLE
added reset structure file for the component library

### PR DESCRIPTION
**Pull request by GuptaEesh**

### CODE REVIEW

File: base/reset.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
+/* Base */
+
+*,
+*::before,
+*::after {
+  border: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+  font-size: var(--size-12);
+  margin: 0 0 0 0;
```
Comment (saadahmadhere) [inline base/reset.css]: margin: 0; would work here imo.

File: base/reset.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+textarea {
+  background-color: var(--primary-100);
+  border-radius: 8px;
+  height: 8rem;
+  width: 30rem;
+}
+
+html .xsm {
```
Comment (saadahmadhere) [inline base/reset.css:27]: these classes would be more fitted under the typography component. Just a suggestion.

**Review by saadahmadhere — APPROVED** [review]
Code looks clean and good to merge.

File: base/reset.css (hunk 2 above)
Comment (GuptaEesh) [reply base/reset.css:27]: In typography I have added more specific classes reset file contains all the generic classes and resets.

File: base/reset.css (hunk 1 above)
Comment (GuptaEesh) [reply base/reset.css]: Yeah changing that😅

Comment (GuptaEesh) [reply base/reset.css]: Done✅

**Review by ZuberDunge — APPROVED** [review]
Clean and clear. avoid extra spacing, rest looks fine. good to go.

**Pull request merged**